ros2node/ros2node/verb/list.py

```diff
@@ -37,4 +37,4 @@ def main(self, *, args):
         if args.count_nodes:
             print(len(node_names))
         elif node_names:
-            print(*[n.full_name for n in node_names], sep='\n')
+            print(*sorted(n.full_name for n in node_names), sep='\n')
```

Comment:
This change as well as the ones in the `bw` module are functional changes not related to testing. Please move them into a separate PR. Otherwise when the PR is being squash merged they are "hidden" in a commit only mentioning "test coverage".

Reply:
Oops, yes, you're right. I'll split it up.

Reply:
See #305 and #306.